scripts/generate_errors.pl

```diff
@@ -46,6 +45,44 @@
 
 $/ = $line_separator;
 
+=comment
```

Comment:
I don't get what the purpose of leaving commented block of code in the script is. Can you explain?